lightning/src/ln/channelmanager.rs

```diff
@@ -16791,6 +16854,7 @@ where
 			pending_intercepted_htlcs: Mutex::new(pending_intercepted_htlcs.unwrap()),
 
 			forward_htlcs: Mutex::new(forward_htlcs),
+			receive_htlcs: Mutex::new(receive_htlcs),
```

Comment:
Earlier in this method, we remove pending payments that are not present in the monitors, and ISTM we should be doing this for `receive_htlcs` now as well